.NET/Microsoft.Recognizers.Text.Sequence/SequenceRecognizer.cs

```diff
@@ -89,7 +90,12 @@ public IModel GetEmailModel(string culture = null, bool fallbackToDefaultCulture
 
         public IModel GetURLModel(string culture = null, bool fallbackToDefaultCulture = true)
         {
-            return GetModel<URLModel>(culture, fallbackToDefaultCulture);
+            if (culture == "zh-cn" || culture == "ko-kr" || culture == "ja-jp")

```

Comment:
I don't think Korean needs this. As far as I know, it uses whitespace as separator everywhere.
Also, this could be an attribute in the language-specific url configs. IsWhitespaceSeparatorCulture. Only Chinese and Japanese would return true.
The attribute could be declared in YAML and exposed through the new config class you added.